pkg/plugin/config.go

```diff
@@ -0,0 +1,10 @@
+package plugin
+
+// PluginConfig contains dynamic plugin configuration
+type Config struct {
+	// The sync image to use
+	SyncImage string `json:"syncImage,omitempty"`
+
+	// The node image to use
+	NodeImage string `json:"nodeImage,omitempty"`
```

Comment:
I would suggest `vmImage` or `machineImage`. `nodeImage` is confusing even though we don't run kubelet inside a container (and there is also `origin-node`).

Reply:
changed to `VmImage`